public/js/office.web.js

```diff
@@ -14,9 +14,15 @@ $(() => {
   const logoutButton = $('#btnLogout');
 
   logoutButton.on('click', (e) => {
+    if(!gapi.auth2){
+      gapi.load('auth2', function() {
```

Comment:
I believe you can use the function onLoad at the bottom of this file.

Reply:
@raphaelln Could you please, check it again? I made some changes.